ctmc_lectures/_config.yml

```diff
@@ -1,6 +1,6 @@
 # Book settings
 title: Continuous Time Markov Chains
-author: John Stachurski
+author: Thomas J. Sargent & John Stachurski
```

Comment:
@mmcky I had pushed the author change in a previous commit https://github.com/jstac/continuous_time_mcs/commit/1233c44c1f5e7b60be44efadf8cb246cebdc8f87

Reply:
I'll merge in master